guide/src/index.html

```diff
@@ -1,7 +1,7 @@
 <html>
     <head>
         <meta charset="utf-8" />
-        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css' />
+        <link href='https://fonts.googleapis.com/css?family=Roboto:300,400' rel='stylesheet' type='text/css' />
```

Comment:
Sorry it was probably a bit hasty to remove. I didn't really investigate, just figured it looked better without.

I'll sort it out what is actually happening and offer a proper solution in another PR.